chrome/updater/updater_module.cc

```diff
@@ -266,10 +266,17 @@ void UpdaterModule::Update() {
     return;
   }
 
+#if defined(COBALT_BUILD_TYPE_GOLD)
+  bool skipVerifyPublicKeyHash = false;
+#else  // defined(COBALT_BUILD_TYPE_GOLD)
+  bool skipVerifyPublicKeyHash = GetAllowSelfSignedBuilds();
```

Comment:
Based on the design I thought we were adding a web API to set a public key to use for verification of the self-signed package. Did we decide instead to skip package verification altogether?

Reply:
Yeah, from the security review I go the impression that they don't see that approach as more secure than skipping the signature check, and it was more difficult for users.

If we agree as a team that we should do that instead I can update this, it would pretty much just be changing this to a string and instead of clearing the key list in the updater_module I'd instead add the given key hash. LMK what you think.

Reply:
That sounds good to me, I also don't see a significant difference in security between the two approaches.

It may be good to just capture that feedback from the security review in the design doc somewhere, maybe under Design Review Notes.

Reply:
The PR lgtm. The current approach is good to me, as long as the security review team and EG team decide that skipping verification is okay. I only have some nit comments. Other than that, please take a look at the failed checks and resolve them. Thanks Tyler!